CHANGELOG.md

```diff
@@ -3,59 +3,90 @@
 
-We thank Harshil Patel ([@drpatelh](https://github.com/drpatelh)) and everyone in the Luscombe Lab ([@luslab](https://github.com/luslab)) for their extensive assistance in the development of this pipeline.
+### Fixes
 
-### Pipeline summary
+- Fixed confusing config warnings that were being displayed on legitmate parameter configurations.
+- Fixed deeptools correlation plots that were showing low levels of correlation even in test data by changing the plot to use the pearson test.
+- Corrected the SEACR p-value parameter description.
+- Fixed output of picard mark/remove duplicate files so that the sorted, indexed bams for all files always output to the results folder.
+- Spikein genome processes and checks no longer run when the normalisation mode is set to something other than `SpikeIn`.
+- Pipeline will now fail gracefully when single-end reads are detected.
 
-1. Check input files
-2. Merge re-sequenced FastQ files ([`cat`](http://www.linfo.org/cat.html))
-3. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-4. Adapter and quality trimming ([`Trim Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
-5. Alignment to both target and spike-in genomes ([`Bowtie 2`](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml))
-6. Filter on quality, sort and index alignments ([`samtools`](https://sourceforge.net/projects/samtools/files/samtools/))
-7. Duplicate read marking ([`picard`](https://broadinstitute.github.io/picard/))
-8. Create bedGraph files ([`bedtools`](https://github.com/arq5x/bedtools2/)
-9. Create bigWig coverage files ([`bedGraphToBigWig`](http://hgdownload.soe.ucsc.edu/admin/exe/))
-10. Peak calling specifically tailored for low background noise experiments ([`SEACR`](https://github.com/FredHutch/SEACR))
-11. Consensus peak merging and reporting ([`bedtools`](https://github.com/arq5x/bedtools2/))
-12. Quality control and analysis:
-    1. Alignment, fragment length and peak analysis and replicate reproducibility ([`python`](https://www.python.org/))
-    2. Heatmap peak analysis ([`deepTools`](https://github.com/deeptools/deepTools/))
-13. Genome browser session ([`IGV`](https://software.broadinstitute.org/software/igv/))
-14. Present QC for raw read, alignment and duplicate reads ([`MultiQC`](http://multiqc.info/))
+### Software dependencies
 
-## [1.1] - 2022-01-05
+Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
 
-### Enhancements & fixes
+| Dependency | Old version | New version |
+| ---------- | ----------- | ----------- |
+| `multiqc`  | 1.14        | 1.16.1      |
+| `samtools` | 2.27.2      | 2.27.4      |
```

Comment:
samtools version is 1.16.1 from 1.15.1, should the top be samtools? 
Not sure what the second line might be, if anything.

Reply:
good spot thanks